FreeRTOS-Plus/Demo/AWS/Device_Defender_Windows_Simulator/Device_Defender_Demo/report_builder.h

```diff
@@ -52,6 +80,8 @@ typedef struct ReportMetrics
     uint32_t ulOpenUdpPortsArrayLength;
     Connection_t * pxEstablishedConnectionsArray;
     uint32_t ulEstablishedConnectionsArrayLength;
+    CustomMetric_t * pxCustomMetricsArray;
+    uint32_t ulCustomMetricsArrayLength;
```

Comment:
Suggestion for a more expressive name for the struct member
```suggestion
    uint32_t ulNumOfCustomMetrics;
```

Reply:
This would make it inconsistent with the rest of the names.